src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,136 @@
+package games.strategy.engine.data;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
```

Comment:
The concept of "schema ID" is actually defined by `PropertyBagExporter` and `PropertyBagImporter`.  Those classes have no knowledge of what property bag memento schemas they will have to deal with.  Therefore, using an enum is probably not possible.  Although, following some of your recent examples, defining a `SchemaId` type that simply wraps a string could be used to provide stronger typing.

In reality, I believe the schema ID was added to help support versioning.  When versioning was removed, I'm not sure why I didn't remove this concept, as well.  A memento parser can't necessarily rely on a matching schema ID to inform it that all required name-value pairs are present--it still must handle missing name-value pairs appropriately.  So I think it's just best to remove the schema ID completely.  (As stated elsewhere, `PropertyBagMemento` may be replaced with a strongly-typed memento; in which case, the strongly-typed memento class name effectively becomes the schema ID.)